package.json

```diff
@@ -29,6 +29,8 @@
     "eslint-plugin-react": "^4.2.3",
     "file-loader": "^0.8.5",
     "ignore-styles": "^1.2.0",
+    "jsdom": "^8.4.0",
+    "jsdom-global": "^1.7.0",
```

Comment:
had to add these deps (plus `--require jsdom-global/register`) to get the Modal code that operates on `window` to run in tests